src/language-js/parse/babel.js

```diff
@@ -74,7 +85,7 @@ function isFlowFile(text, options) {
 
 function parseWithOptions(parseMethod, text, options) {
   // Inline the require to avoid loading all the JS if we don't use it
-  /** @type {import("@babel/parser").parse | import("@babel/parser").parseExpression} */
+  /** @type {Parse} */
```

Comment:
Shouldn't it lowercased?

Reply:
I have no strong opinion. But, generally type names should be upper camel cased in TypeScript. `("@babel/parser").parse` and `import("@babel/parser").parseExpression` are function (not type) so they are lowercased.

Reply:
@fisker What do you think about the above comment?

Reply:
I don't use jsdoc at all, I can't review.